libraries/entities/src/DiffTraversal.cpp

```diff
@@ -137,9 +137,9 @@ void DiffTraversal::Waypoint::getNextVisibleElementDifferential(DiffTraversal::V
                     ++_nextIndex;
                     if (nextElement) {
                         AACube cube = nextElement->getAACube();
-                        if ( view.viewFrustum.calculateCubeKeyholeIntersection(cube) != ViewFrustum::OUTSIDE) {
+                        if (view.viewFrustum.calculateCubeKeyholeIntersection(cube) != ViewFrustum::OUTSIDE) {
                             ViewFrustum::intersection lastIntersection = lastView.viewFrustum.calculateCubeKeyholeIntersection(cube);
-                            if (!(lastIntersection == ViewFrustum::INSIDE && nextElement->getLastChanged() < lastView.startTime)) {
+                            if (lastIntersection != ViewFrustum::INSIDE || nextElement->getLastChanged() > lastView.startTime) {
```

Comment:
Applying Morgan's Theorem because the code looked more readable this way.

Reply:
you flipped the `<` to a `>`....  typically when flipping a boolean evaluation like this `<` becomes `>=`, not `>`.  So is this correct? or should it have been a `<=` before?

Reply:
Good eye.  I chose `<` instead of `<=` because it is, in theory, slightly faster, and in this case it doesn't really matter.  Anything time check that fails here would have been picked up by the previous scan, since that is the exact case we're checking here.

Well, "it doesn't matter" as long as all the other places that check the timestamps do it in the same way: `if (entity->timestamp > view.timestamp) { do_stuff }`.  I will audit the rest of the code and verify/enforce this is the case.

Reply:
I confirm: everything is checking the code in the correct way: `if (element->timestamp > view.timestamp) { do_stuff }`.  Which means by making this change I actually removed a subtle and rare bug which would have kicked in whenever an in-view element's timestamp was exactly at the start of the last completed traversal AND we were performing a differential scan.